dashboard/package.json

```diff
@@ -8,8 +8,8 @@
     "build": "NODE_ENV=production webpack --config src/config/webpack.app.js",
     "lint": "eslint \"src/**\"",
     "start": "hjs-dev-server src/config/webpack.app.js",
-    "test": "wdio test/conf/wdio.local.js",
-    "test-browserstack": "wdio test/conf/wdio.browserstack.js",
+    "test": "wdio test/conf/wdio.local.js --suite base",
+    "testExtended": "EXTENDED=1 wdio test/conf/wdio.local.js",
```

Comment:
Maybe I'm not understanding something here. I assumed that the following will run only the tests in the `extended` suite, which are also the tests that require `EXTENDED=1`:

```
EXTENDED=1 wdio test/conf/wdio.local.js --suite extended
```